### ISSUE TITLE
Space in "Program Files" makes lesson 00 impossible

### ISSUE THREAD

**Issue by cakenggt** [npm/how-to-npm#56]
Running Windows and having a space in your node filepath makes completing lesson 00 impossible since it thinks that the rest of the filepath is an argument. Error text below:

```
verifying that npm is installed...
Uh oh!  npm had a problem! {"killed":false,"code":1,"signal":null,"cmd":"C:\\WIN
DOWS\\system32\\cmd.exe /s /c \"C:\\Program Files\\nodejs\\npm.CMD --version\""}

'C:\Program' is not recognized as an internal or external command,
operable program or batch file.
#########################################
###   YOUR SOLUTION IS NOT CORRECT!   ###
#########################################
```

**Comment by derekharmon** [comment]
Related #11 #17 #19 #32 #33 #57

**Comment by martinheidegger** [comment]
Should be closed with #57 ?

**Comment by cakenggt** [comment]
Yes, thank you!
